test/scale/test-scale.sh

```diff
@@ -489,16 +525,18 @@ if [[ $numUniqueLabelsPerPod -gt 0 ]]; then
     done
 fi
 
-set -x
+wait_for_pods
```

Comment:
to better evaluate time to apply ACLs, wait for pods to come up first (takes a variable amount of time) before applying the NetPols